src/Patterns/FilterPatterns.php

```diff
@@ -0,0 +1,200 @@
+<?php
+/**
+ * Title: Pattern example
+ * Slug: woocommerce-blocks/pattern-example
+ * Categories: WooCommerce
+ * Block Types: core/paragraph
+ */
+
+namespace Automattic\WooCommerce\Blocks\Patterns;
+
+use Automattic\WooCommerce\Blocks\Domain\Package;
+
+/**
+ * FilterPatterns class
+ */
+class FilterPatterns {
```

Comment:
The class name could probably be changed here to just something like `Patterns`? This will handle registering any pattern in the `/patterns` path right?

Reply:
That's right, changing it!